### DISCUSSION TITLE
Image as Text "Color"

### DISCUSSION THREAD

**Discussion by DanielK1511** [tailwindlabs/tailwindcss#9476] [Help] [answered]
Hey People!

So I am trying to have an Image or SVG as Background / Color for a strip of text i have.

As we know, you can do something similar but with gradients: https://play.tailwindcss.com/IUmhN1i1WR

But I want to have it similar to: 
![Unbenannt](https://user-images.githubusercontent.com/36012642/193794820-bdb22a95-9303-4d5d-b018-dc26dbb2b44f.PNG)

Has anybody an idea how to approach this in tailwind?

Looking forward to your replies
Daniel

**Comment by wongjn** [comment, marked as answer]
In CSS, using an SVG is a similar way to how you would use a gradient, since they both are applied using the `background-image` property. In Tailwind, you would most likely use an [arbitrary value](https://tailwindcss.com/docs/background-image#arbitrary-values) for the SVG. [Here's an example of using an arbitrary value](https://play.tailwindcss.com/FyX75Zu89n).

**Reply by DanielK1511** [reply]
Thank you so much!

**Comment by DanielK1511** [comment]
This is for my own usecase, u can ignore.


`<h1 class="w-max bg-[url(https://app.gitbook.com/public/app/assets/default-page-cover-K4LPTE6Y.png)] bg-cover bg-clip-text text-5xl font-bold text-transparent">Test</h1>`
